src/imitation/scripts/common/demonstrations.py

```diff
@@ -106,3 +113,19 @@ def load_expert_trajs(
         expert_trajs = expert_trajs[:n_expert_demos]
         logger.info(f"Truncated to {n_expert_demos} expert trajectories")
     return expert_trajs
+
+
+@demonstrations_ingredient.capture(prefix="expert")
+def _download_expert_rollouts(rollout_type, loader_kwargs):
+    if not rollout_type.endswith("-huggingface"):
+        raise ValueError(
+            "`rollout_type` must follow the convention `{algo}-huggingface`"
+            "to download rollouts from huggingface of an expert trained using {algo}."
+            "Example: rollout_type=ppo-huggingface",
+        )
+    algo_name = rollout_type.split("-")[0]
+    return serialize.load_rollouts_from_huggingface(
+        algo_name,
+        env_name=loader_kwargs["env_name"],
+        organization=loader_kwargs["organization"],
```

Comment:
It's a little counterintuitive that it depends on this. I can't think of anything better, it'd be worse to have code duplication, but maybe we want to document this somewhere?

Reply:
Adding an example in the documentation should be good. Although I suspect users would rarely want to change the default `loader_kwargs` since it requires uploading the rollouts in our format to an organization other than HumanCompitableAI.